lib/config.js

```diff
@@ -7,7 +7,12 @@ import path from 'path';
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-const cfg = config['account-http'] = {};
+const cfg = config['account-http'] = {
+  turnstile: {
+    SECRET_KEY: '1x0000000000000000000000000000000AA',
+    url: 'https://challenges.cloudflare.com/turnstile/v0/siteverify'
+  }
```

Comment:
Making some room for other options and for contextualizing "authorization" and its use during registration:

```suggestion
  // account authorization type options
  authorization: {
    turnstile: {
      SECRET_KEY: '1x0000000000000000000000000000000AA',
      url: 'https://challenges.cloudflare.com/turnstile/v0/siteverify'
    }
  },
  // account registration options
  registration: {
    authorizationRequired: 'turnstile'
  }
```